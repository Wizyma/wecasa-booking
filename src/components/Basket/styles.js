import styled from 'styled-components';

export const FixedWrapper = styled.div`
  position: fixed;
  bottom: 10px;
  right: 10px;
  background-color: #0069c0;
  color: white;
  width: 60px;
  height: 60px;
  border-radius: 50px;
  text-align: center;
  display: flex;
  align-items: center;
  justify-content: center;
  -webkit-transition: width 2s, height 2s, border-radius 2s; /* For Safari 3.1 to 6.0 */
  transition: width 2s, height 2s, border-radius 2s;
  :hover {
    width: ${props => props.basket && '200px'};
    height: ${props => props.basket && '200px'};
    border-radius: ${props => props.basket && '5px'};
  }
`;
