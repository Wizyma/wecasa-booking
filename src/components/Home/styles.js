import styled from 'styled-components';

export const Grid = styled.div({
  width: '100%',
  display: 'flex',
  flexWrap: 'wrap',
  position: 'relative',
  justifyContent: 'center',
  textAlign: 'center',
});

export const GridColumn = styled.div({
  width: '300px',
  display: 'flex',
  flexWrap: 'wrap',
  boxShadow: '0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23)',
  margin: '10px',
  padding: '10px',
  height: '150px',
  justifyContent: 'center',
  alignItems: 'center',
});

export const Paragraph = styled.p({
  margin: '2px',
  width: '100%',
});
