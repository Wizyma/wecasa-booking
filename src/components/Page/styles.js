import styled from 'styled-components';

export const Layout = styled.div({
  padding: '0 20px',
  minHeight: '100vh',
  width: 'auto',
  display: 'flex',
});
