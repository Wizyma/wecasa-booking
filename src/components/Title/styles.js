import styled from 'styled-components';

export const StyledTitle = variant =>
  styled[variant]({
    width: '100%',
    fontFamily: 'Open Sans',
  });
