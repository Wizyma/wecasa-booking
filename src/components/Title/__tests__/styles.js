import * as styles from '../styles';

describe('test styles', () => {
  it('should match snapshot', () => {
    expect(styles).toMatchSnapshot();
  });
});
