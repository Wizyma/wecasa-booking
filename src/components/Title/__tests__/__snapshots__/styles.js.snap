// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test styles should match snapshot 1`] = `
Object {
  "StyledTitle": [Function],
}
`;
